packages/astro-ghostcms/src/astro-ghostcms.ts

```diff
@@ -0,0 +1,327 @@
+							),
+						);
+					}
+					addIntegration(sitemap(options.Integrations?.sitemap));
+				} else {
+					if (verbose) {
+						GhostIntegrationLogger.info(
+							c.gray(
+								"@astrojs/sitemap integration already exists, skipping...",
+							),
+						);
+					}
+				}
+				// ASTRO-ROBOTS-TXT
+				if (!hasIntegration("astro-robots-txt")) {
+					if (verbose) {
+						GhostIntegrationLogger.info(
+							c.bold(
+								c.magenta(`Adding ${c.blue("astro-robots-txt")} integration`),
+							),
+						);
+					}
+					addIntegration(robotsTxt(options.Integrations?.robotsTxt));
+				} else {
+					if (verbose) {
+						GhostIntegrationLogger.info(
+							c.gray(
+								"astro-robots-txt integration already exists, skipping...",
+							),
+						);
+					}
+				}
+
+				// Set up default 404 page
+				if (!options.disableDefault404) {
+					if (verbose) {
+						GhostRouteLogger.info(
+							c.bold(c.cyan("Setting up default 404 page")),
+						);
+					}
+					injectRoute({
+						pattern: "/404",
+						entrypoint: `${name}/404.astro`,
+						prerender: true,
+					});
+				} else {
+					if (verbose) {
+						GhostRouteLogger.info(
+							c.gray("Default 404 page is disabled, Skipping..."),
+						);
+					}
+				}
+
+				// Add virtual imports for user configuration
+				addVirtualImports({
+					"virtual:@matthiesenxyz/astro-ghostcms/config": `export default ${JSON.stringify(
+						options,
+					)}`,
+				});
+
+				// Add types for user configuration
+				addDts({
+					name: "@matthiesenxyz/astro-ghostcms/config",
+					content: `declare module "virtual:@matthiesenxyz/astro-ghostcms/config" {
+                        const Config: import("../schemas/userconfig").GhostUserConfig;
+                        export default Config;
+                    }`,
+				});
+			},
+			"astro:config:done": ({ logger }) => {
+				// Configure Loggers
+				const GhostLogger = logger.fork(
+					`${c.bold(c.blue("👻 Astro-GhostCMS"))}${c.gray("/")}${c.green(
+						"CONFIG",
+					)}`,
+				);
+
+				// Log Configuration Complete
+				GhostLogger.info(
+					c.bold(c.green("Integration Setup & Configuration Complete")),
+				);
+			},
+			"astro:server:start": async ({ logger }) => {
+				// Configure Loggers
+				const GhostLogger = logger.fork(
+					`${c.bold(c.blue("👻 Astro-GhostCMS"))}${c.gray("/")}${c.bold(
+						c.green("DEV"),
+					)}`,
+				);
+				const GhostUpdateLogger = logger.fork(
+					`${c.bold(c.blue("👻 Astro-GhostCMS"))}${c.gray("/")}${c.bold(
+						c.green("VERSION CHECK"),
+					)}`,
+				);
+
+				// Start the DEV server
+				GhostLogger.info(
+					c.bold(c.magenta("Running Astro-GhostCMS in Deveopment mode 🚀")),
+				);
+
+				// Check for updates
+
+				// Get the latest version of Astro-GhostCMS
+				const currentNPMVersion = await latestVersion(
+					"@matthiesenxyz/astro-ghostcms",
+				);
+
+				// Get the local version of Astro-GhostCMS
+				const packageJson = await fse.readJson(
```

Comment:
is there a reason for not importing it at the top of the file? if yes, I think you can use `resolve` from `createResolver` for the path

Reply:
This is simply to display any new version during dev-mode only... There is a check for latest version and it compares the local version :D

Reply:
grabs the current installed one from `astro-ghostcms` local version and compares to the latestversion from NPM :P I did actually try to use the resolve() util but that just grabbed the playgrounds package.json ironically...

Reply:
Okay i fixed that issue.  Now using `resolve()` properly 😄